lib/awful/titlebar.lua

```diff
@@ -101,6 +104,7 @@ local function new(c, args)
         c:connect_signal("focus", update_colors)
         c:connect_signal("unfocus", update_colors)
     else
+        args.context = {}
```

Comment:
Well, the context is already created.

Users can cause `awful.titlebar.new()` to be called multiple times on the same titlebars. The first call executes the "true"-part of this if and causes a new drawable to be created. The following calls only end up in this "false" part. No new drawable is created here (at one point I had it create multiple drawables and the results were really bad...).

So no, this line doesn't do what you think it does.

(And yes, I guess it might make sense to explain the above in a comment in the code...)